test/expr/postfix/dot/init_ref_delegation.swift

```diff
@@ -270,10 +270,10 @@ func foo<T: C>(_ x: T, y: T.Type) where T: P {
   var c3a = type(of: x).init() // expected-error{{'required' initializer}}
   var c4a = type(of: x).init(proto: "")
 
-  var ci1 = x.init(required: 0) // expected-error{{'init' is a member of the type; use 'type(of: ...)' to initialize a new object of the same dynamic type}} {{15-15=type(of: }} {{19-19=)}} 
-  var ci2 = x.init(x: 0) // expected-error{{'init' is a member of the type; use 'type(of: ...)' to initialize a new object of the same dynamic type}} {{15-15=type(of: }} {{19-19=)}} 
-  var ci3 = x.init() // expected-error{{'init' is a member of the type; use 'type(of: ...)' to initialize a new object of the same dynamic type}} {{15-15=type(of: }} {{19-19=)}} 
-  var ci4 = x.init(proto: "") // expected-error{{'init' is a member of the type; use 'type(of: ...)' to initialize a new object of the same dynamic type}} {{15-15=type(of: }} {{19-19=)}} 
+  var ci1 = x.init(required: 0) // expected-error{{cannot invoke 'T.init' with an argument list of type '(required: Int)'}} {{15-15=type(of: }} {{19-19=)}} 
```

Comment:
I think previous diagnostics in this block were better, since the actual problem is not related to arguments but to the base of the member reference...

Reply:
It doesn't get flagged as `UR_InstanceMemberOnType` or `UR_TypeMemberOnInstance`